baseplate/sidecars/event_publisher.py

```diff
@@ -161,6 +161,21 @@ def publish(self, payload: SerializedBatch) -> None:
         raise MaxRetriesError("could not sent batch")
 
 
+def build_batch(queue: MessageQueue, batcher: TimeLimitedBatch) -> bytes:
+    while True:
+        message: Optional[bytes]
+        try:
+            message = queue.get(timeout=0.2)
```

Comment:
I think it would be better to either have this timeout as a constant defined at the top of the module for visibility or better yet have this be configurable in the function with a default pointing to the constant.